adapters/cpmstar/cpmstartest/supplemental/explicit-dimensions.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Curious. There doesn't seem to be any logic in this adapter for dimensions, so what are you testing here?

Reply:
This is just to ensure that the normal dimensions plus formats case is shown as a potential usage method to end users and to ensure that we show it as supported.